site/_docs/file_adapter.md

```diff
@@ -326,6 +326,6 @@ sqlline> select distinct deptno from depts;
 
 ## Future improvements
 
-We are continuing to enhance the adapter, and would welcome
+We are continuing to enhance the adapter and would welcome
```

Comment:
Correct both with and without comma, but slightly clearer with it, given that there is an 'and' later in the sentence.